### PR TITLE
Ratings and toast

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my PR for Toasts and Rating Components. Suggest changes if any.

### CODE REVIEW

File: components/ratings/ratings.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    display: flex;
+}
+
+.rating-input {
+    display: none;
+}
+
+.rating-icon {
+    font-size: 2.8rem;
+    color: lightgray;
```
Comment (rutu14) [inline components/ratings/ratings.css]: Color Variable can be assigned.

File: components/ratings/ratings.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.rating-input:checked + .rating-icon {
+    color: gold;
```
Comment (rutu14) [inline components/ratings/ratings.css]: Color Variable can be assigned.

**Review by sanu29 — APPROVED** [review]
All good to proceed with commit according to me. Great work could not find any issues.
Spacing is great as well your naming conventions are meaningful and easy.
File structure is also good. Min styles applied and min lines of code kept in each CSS file.
Great work.

File: components/ratings/ratings.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.ratings-primary .rating-icon{
```
Comment (dev-prabhat) [inline components/ratings/ratings.css:18]: I think here you have to put **( , )** between two classes

**Review by dev-prabhat — APPROVED** [review]
Code is readable and clean

**Review by rutu14 — APPROVED** [review]
The code has related property nicely arranged.

**Pull request merged**